src/crypto/openssl/rsa_public_key.h

```diff
@@ -16,14 +16,17 @@ namespace ccf::crypto
   class RSAPublicKey_OpenSSL : public PublicKey_OpenSSL, public RSAPublicKey
   {
   protected:
-    std::pair<std::vector<uint8_t>, std::vector<uint8_t>>
-    rsa_public_raw_from_jwk(const JsonWebKeyRSAPublic& jwk);
+    std::pair<
+      std::vector<uint8_t>,
+      std::vector<
+        uint8_t>> static rsa_public_raw_from_jwk(const JsonWebKeyRSAPublic&
+                                                   jwk);
```

Comment:
```suggestion
   static std::pair<
      std::vector<uint8_t>,
      std::vector<
        uint8_t>> rsa_public_raw_from_jwk(const JsonWebKeyRSAPublic&
                                                   jwk);
```

We usually do this order, surprised code style doesn't enforce it.